### PR TITLE
migration to v10 of reactflow

### PR DESCRIPTION
**Pull request by AlexFreik**

### CODE REVIEW

File: frontend/components/document/DnDFlow.tsx [hunk 1]
```diff
@@ -1,4 +1,10 @@
-import React, { useState, useRef, useEffect, MouseEvent as ReactMouseEvent } from 'react';
+import React, {
+	useState,
+	useCallback,
+	useRef,
+	useEffect,
+	MouseEvent as ReactMouseEvent,
+} from 'react';
 import dynamic from 'next/dynamic';
```
Comment (roman-timo) [inline frontend/components/document/DnDFlow.tsx:8]: I think you can just "import Background from react-flow.." and delete this one from next.

**Review by roman-timo — CHANGES_REQUESTED** [review]

File: frontend/components/document/DnDFlow.tsx (hunk 1 above)
Comment (AlexFreik) [reply frontend/components/document/DnDFlow.tsx:8]: true, it doesn't result in an error anymore

Comment (AlexFreik) [reply frontend/components/document/DnDFlow.tsx]: fixed

**Review by roman-timo — APPROVED** [review]
Looks ok!

**Pull request merged**